WordPress/Classes/ViewRelated/Site Creation/Shared/ErrorStates/ErrorStateViewController.swift

```diff
@@ -33,5 +33,13 @@ final class ErrorStateViewController: UIViewController {
     override func loadView() {
         super.loadView()
         view = contentView
+        trackError()
+    }
+
+    private func trackError() {
+        var errorProperties = [String: AnyObject]()
+        errorProperties["error_info"] = configuration.title as AnyObject?
```

Comment:
The creation of this dictionary is perfectly valid. A slightly more concise (& immutable) flavor:
```
let errorProperties = [
    "error_info": configuration.title as AnyObject
]
```